src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Circle extends AbstractFigure {
+    private final Color color;
+    private final double radius;
+
+    public Circle(Color color, double radius) {
+        this.color = color;
```

Comment:
No, please make each child of AbstractFigure have the same logic, as this one
```suggestion
    private final double radius;

    public Circle(Color color, double radius) {
        super(color)
```

Reply:
not fixed